cardano-api/src/Cardano/Api/Query.hs

```diff
@@ -176,8 +177,13 @@ newtype SerialisedLedgerState era
 data LedgerState era where
   LedgerState :: ShelleyLedgerEra era ~ ledgerera => Shelley.NewEpochState ledgerera -> LedgerState era
 
-instance (Typeable era, Shelley.TransLedgerState FromCBOR (ShelleyLedgerEra era)) => FromCBOR (LedgerState era) where
-  fromCBOR = LedgerState <$> (fromCBOR :: Decoder s (Shelley.NewEpochState (ShelleyLedgerEra era)))
+decodeLedgerState ::
+  forall s era ledgerera.
+  ( ShelleyLedgerEra era ~ ledgerera,
+    Consensus.ShelleyBasedEra ledgerera
+  ) =>
+  Decoder s (LBS.ByteString -> LedgerState era)
```

Comment:
I am now considering if it would be better to do the cbor-in-cbor encoding in the ledger package itself, so that there could be a FromCBOR instance of LedgerState and thus break less downstream. I'll maybe tackle that on Monday if the others agree.

Reply:
I've talked with @nc6 about this, and we decided that not having these odd cbor-in-cbor FromCBOR instances is probably best. So now I just need to make sure that node is adapted to handle the `Annotator (LedgerState era)`.